draft-lmbdhk-quic-multipath.md

```diff
@@ -132,6 +132,68 @@ Both identifiers have the same value, which is the sequence number of the connec
 non-zero connection ID is used. If the connection ID is zero length, the Packet Number Space
 Identifier is 0, while the Path Identifier is selected on path establishment.
 
+# High-level overview {#overview}
+
+
+The multipath extensions to QUIC proposed in this document enable the simultaneous utilization of 
+different paths to exchange non-probing QUIC frames for a single connection. This contrasts with
+the base QUIC protocol {{QUIC-TRANSPORT}} that includes a connection migration mechanism that
+selects only one path to exchange such frames.
+
+A multipath QUIC connection starts with a QUIC handshake as a regular QUIC connection.
+The peers use the enable_multipath transport parameter during the handshake to
+negotiate the utilization of the multipath capabilities.
+The active_connection_id_limit transport parameter bounds the maximum number of paths
+that can be used during a connection. A multipath QUIC connection is thus an established QUIC
```

Comment:
The active_connection_id_limit transport parameter *binds* the maximum number of paths

Reply:
The active CID limits binds the number of simultaneous paths. Or in fact, the number of simultaneous number spaces. It does not bind the total number of paths. When a path is abandoned, the CID is retired, the peer provides a new CID, and that new CID can be used on a new path.